usage-insights-launch.mdx

```diff
@@ -0,0 +1,113 @@
+---
+title: Introducing usage monitoring
+description: 'Learn about the new Usage feature available in Xata'
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/usage-insights-launch.jpg
+  alt: Xata
+author: Siobhán Meier, Ryan Slade
+date: 03-20-2022
+tags: ['launch-week', 'metrics', 'announcement']
+published: false
+slug: usage-metrics-launch
+---
+
+Today we're excited to announce a brand new feature to [monitor what your using](/docs/usage-metrics) in your Xata workspace! Understandably so, our users have wanted this visibility for both planning and billing purposes. This beta release lets users explore their usage metric data at a workspace level for the first time. This is just the starting point, we have a fun roadmap ahead for how we can improve visibility and provide actionable insights on top of these metrics.
```

Comment:
```suggestion
Today we're excited to announce a brand new feature to [monitor what you're using](/docs/usage-metrics) in your Xata workspace! Understandably so, our users have wanted this visibility for both planning and billing purposes. This beta release lets users explore the data of their usage metrics at a workspace level for the first time. This is just the starting point, we have a fun roadmap ahead for how we can improve visibility and provide actionable insights on top of these metrics.
```

Reply:
One comment around the "monitor what you're using" phrase... it might be a little too vague and should instead be something like "monitor what data resources you're using" or something similar... expand on the "what" :)